tests/kernel/boot_ltp.pm

```diff
@@ -14,16 +14,24 @@ use 5.018;
 use warnings;
 use base 'opensusebasetest';
 use testapi;
+use bootloader_setup 'boot_grub_item';
 use serial_terminal 'select_virtio_console';
 
 sub run {
     my ($self, $tinfo) = @_;
     my $ltp_env    = get_var('LTP_ENV');
     my $cmd_file   = get_var('LTP_COMMAND_FILE') || '';
     my $is_network = $cmd_file =~ m/^\s*(net|net_stress)\./;
+    my $is_ima     = $cmd_file =~ m/^ima$/i;
 
-    # During install_ltp, the second boot may take longer than usual.
-    $self->wait_boot(ready_time => 500);
+    if ($is_ima) {
+        # boot kernel with IMA parameters
+        $self->boot_grub_item(3);
```

Comment:
This still looks very *magic* to me, whatever "3" means in this context but if you are happy with it then keep it :)

Reply:
@okurz 3rd line in grub. It's quite well described at the function itself...

Reply:
I agree with @okurz. However by looking at the function, looks Ok I guess :) If you're expecting as many menu options :+1:

Reply:
@okurz @foursixnine I'm sorry, you're right. ATM `add_custom_grub_entries()` (added in this pull request) doesn't allow to add more than one menu with extra parameter (which might be limitation for the future, I'm not sure how many different grub entries will be needed for IMA tests, which would leed to change in `add_custom_grub_entries()`).
So far is `boot_grub_item()` basically useful for booting this new option. Therefore I can change it to be without parameter and boot this third option, or set `3` as default (if you think this useful to keep selecting it for someone in the future).

Reply:
I would be ok with a default :)

Reply:
OK, I'll change it on Monday. Thanks for comment and review to both.